src/components/UserProfile.tsx

```diff
@@ -1,16 +1,36 @@
-import React from "react";
+import { useEffect, useState } from "react";
 import { useRecoilValue } from "recoil";
 import { Link } from "react-router-dom";
 import Button from "@material-ui/core/Button";
 import { currentUserState } from "../stateManagement/userAuth";
 import { auth } from "../firebase";
+import firebase from "firebase";
 
 export default function UserProfile() {
+  let [organizationName, setOrganizationName] = useState<any>();
   const currentUser = useRecoilValue(currentUserState);
+  var db = firebase.firestore();
+  let email: string | null = "ingen bruker";
```

Comment:
Maybe delete, if this is for test purposes? As the profile page should not even be accessible if there is no user object?